### PR TITLE
Add type-snob extension

### PR DESCRIPTION
**Pull request by sawyerh**
## Description

- Add a new extension to make it easy to copy common typographic characters

## Screencast

screenshot

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/type-snob/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+];
+
+export default function Command() {
+  return (
+    <List isShowingDetail={false}>
+      {characters.map((char) => (
+        <List.Item
+          key={char.label}
+          title={char.value}
+          // detail={<List.Item.Detail markdown={`# ${char.label}\n${char.description ?? ""}`} />}
```
Comment (peduarte) [inline extensions/type-snob/src/index.tsx:81]: Wanna remove this comment, or planing on implementing it? 

We can then also remove `isShowingDetail={false}`

**Review by peduarte — COMMENTED** [review]
Hey man, great stuff! Simple and helpful.

Left a small comment, then good to publish.

**Comment by sawyerh** [comment]
Thanks @peduarte. I think I messed something up when I revoked Raycast permissions from my GitHub account after I opened this PR. Any idea how to get unstuck so I can push a change with your suggestion?

CleanShot 2023-03-25 at 10 32 16@2x

File: extensions/type-snob/src/index.tsx (hunk 1 above)
Comment (sawyerh) [reply extensions/type-snob/src/index.tsx]: Planning to implement in a subsequent version, but I've pushed an update to remove it for now: https://github.com/raycast/extensions/pull/5544/commits/87b41e431719096fad3941120547faaa9c6b2a94

**Comment by peduarte** [comment]
Pinging @mathieudutour to help here 🙏

**Comment by peduarte** [comment]
@sawyerh Hey man, looks like you managed to push after all?

**Comment by sawyerh** [comment]
@peduarte Yep, ready for a re-review. Still have issues with the `publish` command but it's no longer blocking.

**Comment by peduarte** [comment]
@sawyerh PR looks great. Thanks for your contribution. Been a fan of your work since ~2009 (iirc), when you made WP themes.

@mathieudutour hey man, could you please advise on how to solve the error above?

**Comment by sawyerh** [comment]
@peduarte oh wow, small world! I appreciate that, thanks man ❤️‍🔥

**Comment by mathieudutour** [comment]
I believe I answered on slack as well, but for posterity:

> Ah I didn't think someone would revoke the token^^
> I'll fix it in the next release. In the meantime, you should be able to open `open ~/.config/raycast/config.json` and remove the `githubaccesstoken` field and try again

**Pull request merged**